requirements.in

```diff
@@ -0,0 +1,3 @@
+-e git+https://github.com/dimagi/commcare-cloud.git@dmr/prep-for-separate-env#egg=commcare-cloud
+nose
+parameterized
```

Comment:
we could make these extra requirements part of the commcare-cloud setup: https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies

Reply:
Oh, hmm.  I think they're already under the `[test]` extra, so maybe I just need to add `[test]` to the end of the above line. I'll try it out